cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/AbstractOperatorState.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.assembly;
+
+import io.strimzi.api.kafka.model.KafkaRebalance;
+import io.strimzi.api.kafka.model.status.Condition;
+import io.strimzi.api.kafka.operator.assembly.KafkaRebalanceState;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeDiagnosingMatcher;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+
+public class AbstractOperatorState {
+
+
+    public static Matcher<KafkaRebalance> hasState(final KafkaRebalanceState state) {
```

Comment:
Surely thios doesn't require an argument, since it is just an assertion that the conditions exist?
or is this matcher unfinished?

Reply:
It is just an assertion that the conditions exist. I have extracted the conditions into the separate matcher, and forgot to remove it.